themes/pure/pure.theme.bash

```diff
@@ -31,11 +31,18 @@ pure_prompt() {
     ps_root_mark="${red} # ${normal}"
     ps_path="${yellow}\w${normal}";
 
+	# Virtualenv
```

Comment:
The indentation looks wrong - it does not seem to match the rest of the file. Can you please check to make sure you're using the same 4 spaces for indentation, as in the rest of the file?